### PR TITLE
rename package to `org.quiltmc.enigma`

### PR DESCRIPTION
**Pull request by ix0rai**
finally. the real breaking change

also renames packages in the test inputs to match convention, and does some gradle trickery so the task names stay the same

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,4 +26,4 @@ Pre-compiled jars can be found on the [Quilt maven](https://maven.quiltmc.org/re
 
 ### On the command line
 
-`java -cp enigma.jar cuchaz.enigma.command.Main`
+`java -cp enigma.jar command.org.quiltmc.enigma.Main`
```
Comment (IotaBread) [inline README.md:29]: wrong package

File: enigma-swing/src/main/java/org/quiltmc/enigma/gui/elements/MenuBar.java [hunk 2]
```diff
@@ -590,7 +591,8 @@ private static void prepareThemesMenu(JMenu themesMenu, Gui gui) {
 			themeButton.addActionListener(e -> {
 				UiConfig.setLookAndFeel(lookAndFeel);
 				UiConfig.save();
-				ChangeDialog.show(gui.getFrame());
+				Themes.setupTheme();
+				//ChangeDialog.show(gui.getFrame());
```
Comment (IotaBread) [inline enigma-swing/src/main/java/org/quiltmc/enigma/gui/elements/MenuBar.java:595]: ?

Comment (ix0rai) [reply enigma-swing/src/main/java/org/quiltmc/enigma/gui/elements/MenuBar.java:595]: oops, was messing around and forgot to revert it seems

File: README.md (hunk 1 above)
Comment (ix0rai) [reply README.md:29]: intellij had a field day trying to figure out how to rename things

**Review by IotaBread — APPROVED** [review]

**Pull request merged**